githubscript/src/githubscript/script.py

```diff
@@ -16,7 +17,20 @@
 async def async_main(config, task):
     prefix = extract_common_scope_prefix(config, task)
     project = get_github_project(task, prefix)
-    project_config = config["github_projects"][project]
+
+    # TODO: write a unittest for this
+    projects = config["github_projects"].keys()
+    if project in projects:
+        project_config = config["github_projects"][project]
+    else:
+        project_config = {}
+        for p in projects:
+            if p.endswith("*"):
```

Comment:
we probably want something like

```
if not p.endswith("*"):
    continue
cleaned = p.rstrip("*")
```

so we don't iterate over any non-`*` projects in this loop (including the `startswith` check).